app/src/main/res/values/strings.xml

```diff
@@ -2,144 +2,144 @@
+    <string name="Witchspells_Free_Access_Title_Plural">+ %1$d Free Access Spells</string>
 
     <plurals name="Witchspells_Free_Access_Title">
         <item quantity="one">Witchspells_Free_Access_Title_Singular</item>
         <item quantity="other">Witchspells_Free_Access_Title_Plural</item>
     </plurals>
 
-    <string name="Witchspells_Free_Access_Format_Singular">%1$d Sort d\'accès libre</string>
-    <string name="Witchspells_Free_Access_Format_Plural">%1$d Sorts d\'accès libre</string>
+    <string name="Witchspells_Free_Access_Format_Singular">%1$d Free Access Spell</string>
+    <string name="Witchspells_Free_Access_Format_Plural">%1$d Free Access Spells</string>
 
     <plurals name="Witchspells_Free_Access_Format">
         <item quantity="one">@string/Witchspells_Free_Access_Format_Singular</item>
         <item quantity="other">@string/Witchspells_Free_Access_Format_Plural</item>
     </plurals>
 
-    <string name="Witchspells_No_Secondary_Spells">Aucun sort secondaire</string>
+    <string name="Witchspells_No_Secondary_Spells">No Secondary Spells</string>
 
-    <string name="Witchspells_Name_Label">Titulaire du grimoire</string>
+    <string name="Witchspells_Name_Label">Spellbook\'s Holder</string>
 
-    <string name="Witchspells_Choose_Main_Path">Choisir ses voies</string>
-    <string name="Witchspells_Choose_Secondary_Path">Choisir sa voie secondaire</string>
+    <string name="Witchspells_Choose_Main_Path">Choose your main path</string>
+    <string name="Witchspells_Choose_Secondary_Path">Choose your secondary path</string>
 
-    <string name="Witchspells_Major_Path">Voie majeure</string>
-    <string name="Witchspells_Minor_Path">Voie mineure</string>
-    <string name="Witchspells_Opposite_Path_Format">Voie opposée : %1$s</string>
-    <string name="Witchspells_Secondary_Path_Format">Secondaire : %1$s</string>
+    <string name="Witchspells_Major_Path">Major Path</string>
+    <string name="Witchspells_Minor_Path">Minor Path</string>
+    <string name="Witchspells_Opposite_Path_Format">Opposite Path : %1$s</string>
+    <string name="Witchspells_Secondary_Path_Format">Secondary Path : %1$s</string>
 
     <plurals name="Witchspells_Free_Access_Count">
-        <item quantity="one">+ 1 sort d\'accès libre</item>
-        <item quantity="other">+ %d sorts d\'accès libre</item>
+        <item quantity="one">+ 1 free access spell</item>
+        <item quantity="other">+ %d free access spells</item>
     </plurals>
 
-    <string name="Witchspells_Add_Label">Ajouter un grimoire</string>
-    <string name="Error_No_Witchspells_Name">Un nom est nécessaire pour pouvoir enregistrer le grimoire</string>
-    <string name="Witchspells_Save_Waiting_Message">Enregistrement du grimoire…</string>
+    <string name="Witchspells_Add_Label">Add a spellbook</string>
+    <string name="Error_No_Witchspells_Name">You need to a name to be able to save your spellbook</string>
```

Comment:
"A name is needed to create your spellbook"
Il faudrait changer la description en français également, on enregistre pas le grimoire on le crée en fait